atlasdb-cassandra/src/main/java/com/palantir/atlasdb/cassandra/CassandraKeyValueServiceConfig.java

```diff
@@ -241,6 +241,12 @@ public final String type() {
         return TYPE;
     }
 
+    @Override
+    @Value.Default
+    public int maxConcurrentGetRanges() {
+        return poolSize() * servers().size();
```

Comment:
what's the difference between this and `rangesConcurrency` above?

Reply:
`rangesConcurrency` was the maximum concurrency allowed for pre-fetching in the old implementation. This is the maximum thread pool size and the concurrency level is specified by the user. Could probably deprecate the `rangesConcurrency` if that makes sense.

Reply:
i see.. how about calling this `getRangesThreadPoolSize`? a little more descriptive plus helps differentiate it from the old thing